src/applications/disability-benefits/all-claims/content/disabilityLabels.js

```diff
@@ -805,17 +805,17 @@ export default {
   9005: 'Musculoskeletal and Muscle Injuries',
   9006: 'Neurological - Cranial/Peripheral Nerves',
   9007: 'Neurological other System',
-  9008: 'Pension',
-  9009: 'Pension - Aid and Attendance/Housebound',
+  // 9008: 'Pension',
+  // 9009: 'Pension - Aid and Attendance/Housebound',
```

Comment:
@goldenmeanie In the original ticket, you said `9009 Pension Aid & Attendance`, which matches this ID, but not the name exactly. Can you confirm that this should still be removed?

Reply:
Good question, but its fine to remove 9009